tests/lib/rules/no-unused-prop-types.js

```diff
@@ -559,26 +559,6 @@ ruleTester.run('no-unused-prop-types', rule, {
       options: [{customValidators: ['CustomValidator']}]
     }, {
       code: [
-        'class Comp1 extends Component {',
-        '  render() {',
-        '    return <span />;',
-        '  }',
-        '}',
-        'Comp1.propTypes = {',
-        '  prop1: PropTypes.number',
-        '};',
-        'class Comp2 extends Component {',
-        '  render() {',
-        '    return <span />;',
-        '  }',
-        '}',
-        'Comp2.propTypes = {',
-        '  prop2: PropTypes.arrayOf(Comp1.propTypes.prop1)',
```

Comment:
This test implies that Comp1's prop1 prop should be marked as used, because Comp2 references its propType; but in fact I think it shouldn't - could you move this test to be *invalid* instead of just deleting it?